common/flatpak-transaction.c

```diff
@@ -2178,8 +2258,16 @@ add_related (FlatpakTransaction          *self,
         {
           FlatpakRelated *rel = g_ptr_array_index (related, i);
           FlatpakTransactionOperation *related_op;
+          gboolean download = rel->download;
 
-          if (!rel->download)
+          if (!download)
+            {
+              g_autofree char *id = flatpak_decomposed_dup_id (rel->ref);
+              if (g_str_has_suffix (id, ".Debug") && priv->auto_install_debug)
```

Comment:
Maybe a nitpick but it would be slightly more efficient if the `priv->auto_install_debug` check were in the `if` above right?